autokeras/hypermodel/preprocessor.py

```diff
@@ -173,11 +135,28 @@ def transform(self, x):
     def output_types(self):
         return tf.float64,
 
+    @property
     def output_shape(self):
-        return self.mean.shape
-
-
-class TextToIntSequence(HyperPreprocessor):
+        return self.inputs[0].shape
+
+    def get_weights(self):
```

Comment:
These are not "weights" (numerical arrays), so prefer using a different name, e.g. save/load or get_state / set_state